### DISCUSSION TITLE
Quest Time Limits

### DISCUSSION THREAD

**Discussion by sLmpgn** [SmartlyDressedGames/Unturned-3.x-Community#3644] [Ideas]
quest time limits, pretty please.

**Comment by Ongjming** [comment]
List of current conditions that is related to time (Though none of them gives exact time limit from quest start point)
(https://github.com/SmartlyDressedGames/U3-Docs/blob/master/NPCAsset/Conditions.md)
Is_Full_Moon
Holiday
Time_Of_Day
Weather related if custom weather is set to occur periodically.

**Reply by sLmpgn** [reply]
This wasn't necessary. I'm aware of the current NPC Conditions for time. None of them are so simple as "Quest_Time_Limit 300." If I wanted a time limit for a quest, I'd need a plugin.
